blablas/ep24/index.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2019-12-29
+time: 21h
+duration: "2:18:03"
+title: "Geeksblabla MSS with Yassine El Kachchani"
+tags: ["startup", "side hustle", "morocco"]
+isNext: false
+video: "2770524023008256"
+published: true
+url: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+podcastUrl: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+---
+
+In this episode of GeeksBlabla, we discuss startuping with Yassine, the CEO of United Remote.
+
+## Guests
+
+[Yassine El Kachchani](https://www.facebook.com/kachchani)
+
+[Mohammed Aboullaite](https://www.facebook.com/aboullaite)
```

Comment:
Delete Mohammed from Guests and add him to the ` Prepared and Presented by `section.

Reply:
Removed from here: https://github.com/DevC-Casa/geeksblabla.com/pull/73/commits/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fL20-L21

Added here: https://github.com/DevC-Casa/geeksblabla.com/pull/73/commits/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fR86-R87